scipy/stats/_stats_py.py

```diff
@@ -7633,7 +7633,6 @@ def _compute_prob_outside_square(n, h):
 
 def _count_paths_outside_method(m, n, g, h):
     """Count the number of paths that pass outside the specified diagonal.
```

Comment:
Let's not make unnecessary whitespace changes.
```suggestion
    """Count the number of paths that pass outside the specified diagonal.

```